multi-agent/lib/mas/sharing/service.py

```
@@ -94,11 +104,48 @@ def accept_invite(self, share_id: str, *, recipient_user_id: str) -> ShareResult
+                )
+            else:
+                raise ValueError(f"Unknown item kind: {item_kind}")
+
+            return ShareResult(
+                share_id="",
+                new_item_id=new_item_id,
+                rid_mapping=rid_mapping,
+                created_resources=len(rid_mapping),
+                name_conflicts=name_conflicts
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)